src/orders/components/OrderCustomerAddressesEditDialog/OrderCustomerAddressesEditDialog.stories.tsx

```diff
@@ -5,39 +5,57 @@ import React from "react";
 import { countries, order as orderFixture } from "../../fixtures";
 import OrderCustomerAddressesEditDialog, {
   OrderCustomerAddressesEditDialogProps
-} from "./OrderCustomerAddressesEditDialog";
+} from ".";
+import { AddressEditDialogVariant } from "./types";
 
 const order = orderFixture("");
 
 const props: OrderCustomerAddressesEditDialogProps = {
   confirmButtonState: "default",
+  variant: AddressEditDialogVariant.CHANGE_CUSTOMER,
   loading: false,
   onClose: () => undefined,
   onConfirm: () => undefined,
   open: true,
-  errors: undefined
+  errors: undefined,
+  countries
 };
 
-storiesOf("Orders / OrderCustomerAddressesEditDialog", module)
+storiesOf("Orders / Changing address in order", module)
   .addDecorator(Decorator)
-  .add("default", () => (
+  .add("Address change when customer is changed", () => (
```

Comment:
Another nitpick but every other story at this level is written in lowercase 🤷‍♂️